packages/types/custom/core.ts

```diff
@@ -431,6 +431,10 @@ export type ToastTypeStyles = {
 /** configuration for toast type styles */
 export type ToastConfiguration = PluginOptions & ToastTypeStyles
 
+export interface ScaleConfiguration {
```

Comment:
Just like with every other configuration option, this needs to extend `PluginOptions`.

Also, I've missed that in `packages/plugins/Scale/src/index.ts` the type needs to be adjusted to `ScaleConfiguration` as well.

Reply:
6ffeaeac8721fd673e2156164c8a500b816f7993

Reply:
1b41081c579aa08e44b16df93e50f38127afe9e6